pkg/ccl/logictestccl/tests/local/local_test: TestCCLLogic_triggers_explain failed

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_triggers_explain [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17961030?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/17961030?buildTab=artifacts#/) on master @ [d43e96e4b81c9782c0af3566c579e7ffdd3404c5](https://github.com/cockroachdb/cockroach/commits/d43e96e4b81c9782c0af3566c579e7ffdd3404c5):


```
                    │                                 regions: <hidden>
                    │                                 actual row count: 1
                    │                                 vectorized batch count: 0
                    │                                 KV time: 0µs
                    │                                 KV contention time: 0µs
                    │                                 KV rows decoded: 1
                    │                                 KV pairs read: 2
                    │                                 KV bytes read: 8 B
                    │                                 KV gRPC calls: 1
                    │                                 estimated max memory allocated: 0 B
                    │                                 MVCC step count (ext/int): 0/0
                    │                                 MVCC seek count (ext/int): 0/0
                    │                                 estimated row count: 1,000 (missing stats)
                    │                                 table: child@child_pkey
                    │                                 spans: FULL SCAN
                    │
                    └── • after-triggers
                        │ trigger: t2
                        │
                        └── • render
                            │ columns: (f, k_old, fk_old, old, new)
                            │ render f: f(NULL, old, 't2', 'AFTER', 'ROW', 'DELETE', 111, 'child', 'child', 'public', 0, ARRAY[])
                            │ render k_old: k_old
                            │ render fk_old: fk_old
                            │ render old: old
                            │ render new: new
                            │
                            └── • render
                                │ columns: (new, old, k_old, fk_old)
                                │ render new: NULL
                                │ render old: ((k, fk) AS k, fk)
                                │ render k_old: k
                                │ render fk_old: fk
                                │
                                └── • project
                                    │ columns: (k, fk)
                                    │
                                    └── • scan buffer
                                          columns: (k, fk, old, f, "check-rows")
                                          sql nodes: <hidden>
                                          regions: <hidden>
                                          actual row count: 1
                                          vectorized batch count: 0
                                          estimated row count: 1
                                          label: buffer 1000000
            
    logic.go:2273: 
         /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/29534/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/triggers_explain:1141: too many errors encountered, skipping the rest of the input
[22:42:31] --- done: /home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/29534/execroot/com_github_cockroachdb_cockroach/bazel-out/aarch64-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/triggers_explain with config local: 32 tests, 2 failures
    --- FAIL: TestCCLLogic_triggers_explain/explain_cascade_trigger (0.26s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_triggers_explain.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-45005

--- a/pkg/ccl/logictestccl/testdata/logic_test/triggers_explain
+++ b/pkg/ccl/logictestccl/testdata/logic_test/triggers_explain
@@ -993,6 +993,12 @@ quality of service: regular
                               estimated row count: 1
                               label: buffer 1000000
 
+
+# Ensure that the intent on 'parent' is resolved so that we don't see any
+# contention time reported for 'delete range' below.
+statement ok
+SELECT count(*) FROM parent
+
 query T
 EXPLAIN ANALYZE (VERBOSE) DELETE FROM parent WHERE k = 2;
 ----

